### DISCUSSION TITLE
Welcome to ramalama Discussions!

### DISCUSSION THREAD

**Discussion by rhatdan** [containers/ramalama#81] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ericcurtin** [comment]
I'm Eric, I work on a wide variety of things I guess... I started playing around with AI tools like instructlab, ollama, LocalAI, etc. And then I started writing this tool because I thought we could create a boring inferencing tool, that's easy to use and community-friendly... First I wrote something badly named as podman-ollama, then I wrote another thing badly named as podman-llm... ramalama became the rewrite of podman-llm in python and is much further along than any of the other projects got to at this point...
